MiqoCraftCore/MiqoCraftCore.cs

```diff
@@ -570,14 +584,20 @@ public static string GenerateScenario(List<FFXIVItem> iItemToGenerate, MiqobotSc
                                 fullScenario += "changeJob(" + GarlandTool.GetGatheringJobName(gatheringType, iItem) + ")" + Environment.NewLine;
                                 fullScenario += "selectGrid(" + gridName + ")" + Environment.NewLine;
                                 fullScenario += "selectGatherPreset(" + presetName + ")" + Environment.NewLine;
-                                if (null != gatheredItem && gatheredItem.AsCollectable)
+                                //Special Rotation for Shards Crystal Clusters
+                                if (Namedcatalysts.Contains(iItem.Name))
```

Comment:
Compare if its a Shard/Crystal/Cluster and enforce a rotation with Twelve and if gp the big one with CD (dont remember the name now)